src/theme/components/template.ts

```diff
@@ -1,103 +1,75 @@
-const TemplateBreakout = defineStyleConfig({
-  baseStyle: defineStyle({
-    width: "100%",
-    // This could be "1 / 4" and it would mean the same. This is
-    // "future-proof" the grid column assignment to the last column.
-    gridColumn: "1 / -1",
-  }),
-});
+
 const TemplateContent = defineStyleConfig({
   baseStyle: defineStyle({
-    // Set this element to start on the second 1280px grid column.
-    gridColumn: "2",
-    // This element also contains its own grid system within, but we use "flex"
-    // for mobile to deal with overflow issues related to the Table component.
-    display: { base: "flex", md: "grid" },
-    flexDirection: { base: "column", md: null },
-    gridTemplateColumns: "1fr",
-    paddingY: 0,
-    paddingX: "s",
-    gap: "grid.l",
+    display: "grid",
+    gridTemplateColumns: {
+      base: "repeat(1, minmax(100px, 1fr))",
+    },
+    gap: responsiveGap,
+    padding: responsivePadding,
   }),
-  // With left or right sidebars, we need to set two grid columns and
-  // the column for the sidebar is max 255px width.
+
   variants: {
+    /** If neither `left` nor `right` variants are used, we default to `none` and
+     * the grid will show 1 column only at any viewport width. */
     left: {
-      gridTemplateColumns: { md: "255px 1fr" },
+      gridTemplateColumns: {
+        md: "repeat(2, minmax(100px, 1fr))",
+        lg: "1fr 2fr",
+        xl: "1fr 3fr",
+      },
     },
     right: {
-      gridTemplateColumns: { md: "1fr 255px" },
+      gridTemplateColumns: {
+        md: "repeat(2, minmax(100px, 1fr))",
+        lg: "2fr 1fr",
+        xl: "3fr 1fr",
+      },
     },
   },
 });
 
 const TemplateContentTopBottom = defineStyleConfig({
   baseStyle: defineStyle({
-    gridColumn: { base: "1", md: "1 / span 2" },
-    height: "100%",
+    /** 1 / -1 ensures the item spans all columns, no matter how many there are. */
+    gridColumn: "1 / -1",
   }),
 });
 
-/** The overflow styles were added to deal with overflow issues related to the
- * Table component. */
 const TemplateContentPrimary = defineStyleConfig({
   baseStyle: defineStyle({
-    gridColumn: { base: "1", md: "1 / span 2" },
-  }),
-  variants: {
-    left: {
-      gridColumn: { base: "1", md: "2" },
-      marginEnd: { md: 0 },
-      minWidth: { md: 0 },
-      overflow: { base: "unset", md: "hidden" },
-    },
-    right: {
-      gridColumn: { base: "1", md: "1" },
-      overflow: { base: "unset", md: "hidden" },
-    },
-  },
-});
-const TemplateContentSidebar = defineStyleConfig({
-  variants: {
-    left: {
-      gridColumn: "1",
-    },
-    right: {
-      gridColumn: { base: "1", md: "2" },
+    overflow: "hidden",
+    /**  We need `overflow: hidden` to ensure larger elements passed to the template
+     * appear correctly, but it also cuts off the default focus ring. The following
+     * offset property ensures the focus ring is fully visible. */
+    "*": {
+      _focus: {
+        outlineOffset: "-2px !important",
```

Comment:
Two things with this:
- It works but the focus ring is much closer to every focusable element now. This is a change outside the scope of the template and grid update. We'd have to all be okay with this update independently of the grid/template update.
- This might also work better as a global rule for all elements instead of declaring it here (e.g. set it in the global.ts file)

Reply:
Right, any changes to the focus styles should go through UX exploration first with an aim toward updating the style for all elements. That said, I don't feel a change to the focus style is the way to go with this. There must be a better way to handle the overflow issues, whether that means updating the styles on `Template` or on `Table`.

Reply:
I removed this. It seems (?) to be working now without it, but would like someone else to check and make sure it looks that way to them too.

Reply:
@oliviawongnyc Can you push those upates?